tests/test_edgeql_explain.py

```diff
@@ -488,3 +488,39 @@ async def test_edgeql_explain_type_intersect_01(self):
         }
 
         self.assert_plan(res['Plan'], shape)
+
+    async def test_edgeql_explain_insert_01(self):
```

Comment:
Tests run in transactions by default, so this test is still testing the "already in a transaction" flow. 

To test not in a transaction, the test suite needs to be marked `TRANSACTION_ISOLATION = False`. It's probably not worth creating a new test suite for this, though (since it requires a new database to be created, which is slowish), so maybe just wedge it into TestServerProto or something

Reply:
Ahhh right - let me create a ad-hoc connection that is not affected by the transaction isolation

Reply:
Oh even better